modules/social_features/social_activity/social_activity.tokens.inc

```diff
@@ -14,14 +14,19 @@ use Drupal\node\NodeInterface;
  * Implements hook_token_info().
  */
 function social_activity_token_info() {
-  $additional_information['additional_information'] = [
```

Comment:
in order to be backwards compatible its better to deprecate this token instead of removing it, we will update our own message templates and stop using this token but if other users have this token still in a custom template it will break for them.

Reply:
@ronaldtebrake This means we have to restore, the changes in **social_activity_token_info** and in twig templates.

Reply:
Added changes back https://github.com/goalgorilla/open_social/pull/2277/commits/ce80fd733a58cd4e72f6f6076a52fba307052aff